README.md

```diff
@@ -6,16 +6,20 @@ This package is distributed through [npm](https://www.npmjs.com/).
 
 ## File structure
 
-There are currently two directories, base and utilities. They are both intended to be used on any project but the key difference is how the code is used.
+There are currently three directories, config, variables and utilities. They are intended to be used on any project but the key difference is how the code is used.
 
-### Base
+### Config
 
-Files in the base directory contain CSS which isn't targeted directly using a class but is compiled into the final CSS output (eg box-sizing or reset). It also contains usful mixins such as breakpoints.
+Files in the config directory contain CSS which isn't targeted directly using a class but is compiled into the final CSS output (eg box-sizing or reset). It also contains usful mixins such as breakpoints.
 
 ### Utitilities
 
 Files in utilities contains useful, compact snippets of code which can be accessed directly using a class in your HTML
 
+### Variables
+
+Files in variables contain specific variables for breakpoints and fonts.
```

Comment:
contains* 
for breakpoints, fonts, etc.*

Reply:
Done - sort grammar out.